### PR TITLE
Add quiz to Time Complexity module

### PR DESCRIPTION
**Pull request by ryanchou-dev**
i only have 3 questions rn, if you have any ideas, lmk :D

### CODE REVIEW

File: content/2_Bronze/Time_Comp.mdx [hunk 1]
```diff
@@ -451,3 +452,74 @@ triplets.
...
+    <Quiz.Answer>
+      Measuring the amount of memory an algorithm consumes.
+      <Quiz.Explanation>
+        This is known as space complexity.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      Measuring the number of operations an algorithm performs.
+      <Quiz.Explanation>
+        You'd always want your algorithm to run as fast as possible!
```
Comment (caoash) [inline content/2_Bronze/Time_Comp.mdx:469]: Is there any indicator of a correct answer? If not, maybe change it to "Correct. You'd always want your algorithm..."

File: content/2_Bronze/Time_Comp.mdx [hunk 2]
```diff
@@ -451,3 +452,74 @@ triplets.
...
+    </Quiz.Answer>
+    <Quiz.Answer>
+      The time it takes you to solve a problem.
+      <Quiz.Explanation>
+        
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    What time complexity would a mathematical formula take?
```
Comment (caoash) [inline content/2_Bronze/Time_Comp.mdx:480]: I mean this isn't true for all mathematical formulas (for example fast expo). Maybe be more specific (what time complexity would addition take).

File: content/2_Bronze/Time_Comp.mdx [hunk 3]
```diff
@@ -451,3 +452,74 @@ triplets.
...
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      The worst case scenario.
+    </Quiz.Answer>
+    <Quiz.Answer>
+      The average scenario.
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    What's the time complexity of a for loop?
```
Comment (caoash) [inline content/2_Bronze/Time_Comp.mdx:513]: change to "a for loop from 1 to N" or say "What's the complexity of a for loop through N elements," its not clear what N is here

File: content/2_Bronze/Time_Comp.mdx [hunk 4]
```diff
@@ -451,3 +452,74 @@ triplets.
...
+    <Quiz.Answer correct>
+      $\mathcal{O}(1)$
+      <Quiz.Explanation>
+        No matter how big a certain number is, a mathematical formula would execute the same number of operations.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $\mathcal{O}(N)$
+      <Quiz.Explanation>
+        Would this algorithm take more steps because of a larger number?
```
Comment (caoash) [inline content/2_Bronze/Time_Comp.mdx:490]: technically for something like addition it does (for example, big int addition is slower) but i don't think it matters that much

**Review by caoash — CHANGES_REQUESTED** [review]

File: content/2_Bronze/Time_Comp.mdx (hunk 4 above)
Comment (ryanchou-dev) [reply content/2_Bronze/Time_Comp.mdx:490]: ig big int addition is slower, but it doesn't take require any more calculations, right?
should i change steps to like calculations or smth?

**Comment by nathangong** [comment]
image
is answer choice 1 an error?

**Comment by ryanchou-dev** [comment]
oops misplaced

File: content/2_Bronze/Time_Comp.mdx (hunk 4 above)
Comment (caoash) [reply content/2_Bronze/Time_Comp.mdx:490]: ignore this, it doesn't really matter

**Comment by caoash** [comment]
Looks good.

**Comment by nathangong** [comment]
Wait question 3 also has "correct" in front of the right answer choice

**Comment by ryanchou-dev** [comment]
oops I’ll create a new pr

**Comment by caoash** [comment]
WHOOPS

**Pull request merged**